README.md

```diff
@@ -279,38 +279,10 @@ The unit_test makefile target is used for running the unit tests:
 
 Please have a look to the section [on building the source code](#from-sources) in order to get more information about how to prepare the environment to run the unit_test target.
 
-## Advanced topics:
-
-* Installation and administration
```

Comment:
The idea I had is to link the README.md with the manuals at readthedocs.org ... we can chat about it when you are back, though We need to fix this in order to give guidelines to the rest of the project.